CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Security
+- Updated gogo/protobuf to v1.3.2 to address CVE-2021-3121 (PR TBD)
```

Comment:
Is there an issue to link to?

Reply:
There is not - we haven't really ever put issues in for dependency version bumps like this. I will push up an updated version that links to the PR shortly though.